src/main/java/teammates/ui/webapi/CreateMockResponseRecordAction.java

```diff
@@ -0,0 +1,22 @@
+package teammates.ui.webapi;
+
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+
+/**
+ * Create mock data for response records.
+ */
+public class CreateMockResponseRecordAction extends AdminOnlyAction {
```

Comment:
I don't think this is necessary, given that running the cron task via api does exactly this.